tests/ecmp/test_fgnhg.py

```diff
@@ -166,35 +190,34 @@ def fg_ecmp(ptfhost, duthost, router_mac, net_ports, port_list, ip_to_port, bank
     log_file = "/tmp/fg_ecmp_test.FgEcmpTest.{}.create_flows.log".format(test_time)
 
     exp_flow_count = {}
-    flows_per_nh = NUM_FLOWS/len(port_list)
+    flows_per_nh = NUM_FLOWS / len(port_list)
     for port in port_list:
         exp_flow_count[port] = flows_per_nh
 
     ptf_runner(ptfhost,
-            "ptftests",
-            "fg_ecmp_test.FgEcmpTest",
-            platform_dir="ptftests",
-            params={"test_case": 'create_flows',
-                "exp_flow_count": exp_flow_count,
-                "config_file": FG_ECMP_CFG},
-            qlen=1000,
-            log_file=log_file)
-
+               "ptftests",
+               "fg_ecmp_test.FgEcmpTest",
+               platform_dir="ptftests",
+               params={"test_case": 'create_flows',
+                       "exp_flow_count": exp_flow_count,
+                       "config_file": FG_ECMP_CFG},
+               qlen=1000,
+               log_file=log_file)
 
     log_file = "/tmp/fg_ecmp_test.FgEcmpTest.{}.initial_hash_check.log".format(test_time)
 
     ptf_runner(ptfhost,
-            "ptftests",
-            "fg_ecmp_test.FgEcmpTest",
-            platform_dir="ptftests",
-            params={"test_case": 'initial_hash_check',
-                "exp_flow_count": exp_flow_count,
-                "config_file": FG_ECMP_CFG},
-            qlen=1000,
-            log_file=log_file) 
+               "ptftests",
```

Comment:
I see multiple spacing changes throughout the file, is this because of some IDE? I would prefer that we keep the original formats unless there are spacing issues in the original file.

Reply:
Spaces have been reverted